Cargo.toml

```diff
@@ -67,6 +66,7 @@ atomic-write-file = "0.1.4"
 uuid = { version = "1.9.1", features = ["serde", "v4"] }
 pot = "3.0.0"
 ansi_term = "0.12.1"
+steam-rs = { git = "https://github.com/Bash-09/steam-rs" } # Steam API
```

Comment:
Why are we using a fork instead of [the original version](https://github.com/garhow/steam-rs)?